imagetest/test_suites/image_validation/license_test.go

```diff
@@ -174,7 +175,11 @@ func TestArePackagesLegal(t *testing.T) {
 				t.Fatalf(err.Error())
 			}
 			if !isLegal {
-				t.Fatalf("Found illegal package: %v", filename)
+				if filename == "/usr/share/licenses/perl-Module-Build/LICENSE" {
```

Comment:
this is not appropriate. if you really want to second guess the entire thing and make this license OK, add 'FULLTEXT' to the license names variable.

Reply:
That seems weird because "FULLTEXT" is not actually a license but sure that will work too (although as I stated before, the license names variable doesn't work).